applications/serial_lte_modem/boards/nrf9151dk_nrf9151_ns.overlay

```diff
@@ -4,4 +4,4 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
- #include "nrf9161dk_nrf9161_ns.overlay"
+ #include "nrf9151dk_nrf9151_ns.overlay"
```

Comment:
I think this will break.
Point is to include 9161DK overlay, as these two boards are almost identical.

Don't change that to circular include.